CHANGES/830.misc

```diff
@@ -0,0 +1 @@
+Mark API strings for translation and add 'from django.utils.translation import gettext_lazy as _'
```

Comment:
Suggest renaming pr title to `Mark API strings for translation` to match part of this changelog msg. We typically reuse pr title for commit title when squashing and merging